vendor/github.com/longhorn/backupstore/deltablock.go

```diff
@@ -502,20 +509,17 @@ func RestoreDeltaBlockBackupIncrementally(config *DeltaRestoreConfig) error {
 		return fmt.Errorf("invalid parameter lastBackupName %v", lastBackupName)
 	}
 
-	// check volDev
```

Comment:
This modification is not from me, it was from [this commit](https://github.com/longhorn/backupstore/commit/31a628633b8f0c6ec1abac929d6763d36b5ec1c4)